### ISSUE TITLE
Internl Server Error when using notification condition

### ISSUE THREAD

**Issue by mark-chief** [verbb/formie#292]
**Description**
The below error occurs on form complete, when a notification has a conditional set. If I disable that condition then it works.

Log Error:
[error][yii\base\ErrorException:8] yii\base\ErrorException: Array to string conversion in  /home/runcloud/webapps/app/web/craft/vendor/verbb/formie/src/services/Notifications.php:379

Condition settings. The field is a checkbox field. Condition is matched when checkbox is checked:
<img width="1317" alt="Screenshot 2021-01-12 at 17 24 12" src="https://user-images.githubusercontent.com/18699249/104349562-ff7ef680-54fa-11eb-8cbb-b956c7881034.png">

### Form settings
Please pick from the options below.

- Multi-page form:  No
- Submission Method: Either
- Client-side Validation: Yes or No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.9
- Craft version: 3.5.14
- Multi-site:  No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.10](https://github.com/verbb/formie/releases/tag/1.3.10)
